tsl/test/shared/sql/include/shared_setup.sql

```diff
@@ -6,6 +6,8 @@ SET client_min_messages TO ERROR;
 
 SET ROLE :ROLE_DEFAULT_PERM_USER;
 
+CREATE SCHEMA test;
```

Comment:
why are we adding this?

Reply:
timestamp_limits test uses this and on normal regresscheck we precreate that schema too